content/security-humanops.rst

```diff
@@ -0,0 +1,231 @@
+
+**Enable multi-factor authentication and valid back up** in other services used by SFTPPlus.
+
+
+Email
+=====
+
+
+Enable S/MIME in Emails
+-----------------------
+
+There are free email certificates available.  
+
+A list of S/MIME email certificates that you can use is at http://kb.mozillazine.org/Getting_an_SMIME_certificate
+
+You can use the Comodo email certificate. To obtain a free email certificate from Comodo, go to https://www.instantssl.com/ssl-certificate-products/free-email-certificate.html and sign up to recieve the certificate in your Pro:Atria email. To obtain the Comodo cert in PKCS#12 format, collect the cert in the confirmation email, open Mozilla and go to Advanced > Certificates > View Certificates and backup the cert to obtain the PKCS#12 format.
+
+If you use and recommend another provider please add details in this section.
+
+Once obtained, you can add the digital certificate to your email provider of choice.
+
+
+I want to use PGP/MIME?
+-----------------------
+
+You cannot use both S/MIME and PGP/MIME at the same time.
+PGP/MIME really only makes sense in certain situations.  The general use one is S/MIME.
+
+
+Other email tips
+----------------
+
+Add the Pro:Atria Email Signature Template in your emails.
+
+Set your email settings to prevent HTML/images loading and to block remote content.  
+
+Utilize a blacklist of known spam senders.
+
+
+Use of group emails
+-------------------
+
+Do not add team members that are still on trial to the group until such addition is verified by the team lead.
+
+Do not use team group emails for non SFTPPlus purposes like personal and industry newsletters, announcements, and so on.
+
+There may be some cases where we need to use the group email for related services. When using team group emails for SFTPPlus testing or to open SFTPPlus accounts please notify the group email first that the testing / account is legitimate.  That way, when a test is sent to a team account it is not immediately assumed that it is legitimate. 
+
+
+Phishing and Social Engineering
+===============================
+
+If a phishing attack is encountered or you suspect a phishing attack is encountered notify the team.  
+
+Phishing is also utilized as part of active information gathering while enumerating the target and also points to some form of information gathering/enumerating is taking place.
+
+If encountering potential SE (social engineering) attacks notify the team lead.
+
+
+Passwords
+=========
+
+At some point you will need to use services that are reliant on passwords.
+
+If possible, use non-password options.
+
+Use password management tools (ie LastPass, 1Password, etc) and ensure to enable MFA for these tools.
+
+Do not reuse passwords from your personal accounts with SFTPPlus accounts.
+
+Do not send passwords via cleartext or even better - continue to use services that send passwords via cleartext.  
```

Comment:
I got this wrong on first read. Perhaps _or even better: **do not** continue to_?